docs/source/conf.py

```diff
@@ -41,15 +60,42 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = []
 
+# The suffix of source filenames.
+source_suffix = '.rst'
 
-# -- Options for HTML output -------------------------------------------------
+# The encoding of source files.
+source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = 'sphinx'
 
+# -- Options for HTML output -------------------------------------------------
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-#
-html_theme = 'alabaster'
+
+html_theme = "sphinx_rtd_theme"
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
```

Comment:
Looks like déjà vu, I guess GitHub is telling us that there should be a new line after this. Howver, I don't think this should be an issue as the above line is being deleted.

Reply:
What is the issue here? This is not at the end of the file though right?